src/Compilers/CSharp/Test/Emit/CodeGen/SwitchTests.cs

```diff
@@ -10177,162 +10171,156 @@ static int M(float d)
                 expectedOutput: expectedOutput);
             compVerifier.VerifyIL("C.M", @"
     {
-      // Code size      388 (0x184)
+      // Code size      374 (0x176)
```

Comment:
I find it very interesting that the IL is smaller in both of the modified tests. Do we understand that?